consent-ws/src/main/java/org/broadinstitute/consent/http/cloudstore/GCSStore.java

```diff
@@ -156,6 +157,13 @@ public String putStorageDocument(InputStream stream, String type, String fileNam
         return url.toString();
     }
 
+    public Bucket getBucketMetadata() throws IOException, GeneralSecurityException {
```

Comment:
This method is mostly a wrapper for a Google API; I feel like it would be more helpful to do something like Ontology's `ElasticSearchHealthCheckTest`, which verifies the business logic of the health check itself. Thoughts?